### PR TITLE
frontend: make locize-pull

### PR DESCRIPTION
**Pull request by benma**

### CODE REVIEW

**Review by thisconnect — APPROVED** [review]
LGTM

**Comment by benma** [comment]
@thisconnect i pushed new keys and pulled again here, ptal

File: frontends/web/src/locales/en/app.json [hunk 1]
```diff
@@ -136,7 +136,7 @@
       "understand": "I understand that an incorrect recovery password will create a different wallet"
     },
     "showMnemonic": {
-      "description": "You will be presented with your recovery words, which form a backup of your wallet. Write them down on paper.\n\n<strong>Do not store them digitally or take pictures of it.</strong>\n\n<strong>Do not say the words out loud.</strong>\n\n<strong>This backup is not password-protected.</strong>\n\nAfterwards, you will be asked to confirm each word.",
+      "description": "You will be presented with 24 recovery words, which form a backup of your wallet. Write them down on paper.\n\n<strong>Do not store them digitally or take pictures of it.</strong>\n\n<strong>Do not say the words out loud.</strong>\n\n<strong>This backup is not password-protected.</strong>\n\nAfterwards, you will be asked to confirm each word.",
```
Comment (thisconnect) [inline frontends/web/src/locales/en/app.json:139]: I forgot to update 🏃

Comment (thisconnect) [reply frontends/web/src/locales/en/app.json:139]: @benma please pull again 😇

Comment (benma) [reply frontends/web/src/locales/en/app.json]: done - also pushed 27 new keys from the settings PR.

Comment (thisconnect) [reply frontends/web/src/locales/en/app.json]: thank you

**Pull request merged**